Testing/SystemTests/tests/framework/LoadCIFTest.py

```diff
@@ -14,8 +14,11 @@ class LoadCIFDataWithTwoSectionsTest(systemtesting.MantidSystemTest):
     Tests that the LoadCIF algorithm embedded in the PointCharge class will load cif data which contains two sections,
     and calculates the correct parameters from this data.
     """
-    expected_parameter_values = [-1.004456, -0.488169, 0.579923, 0.000888, -0.001776, 0.000195, 0.001026, 0.000512,
-                                 -5.418847e-7, -4.066401e-7, -1.459750e-7, 2.347822e-7, -3.138693e-7, -1.384160e-6]
+    expected_parameter_values = [
+        -1.324107e+00, -4.179778e-01, 7.644744e-01, 8.222560e-04, -1.914738e-03, 1.713388e-04,
```

Comment:
Not sure why these numbers changed so much

Reply:
It seems to be to do with a compounding effect in [pointcharge.py](https://github.com/mantidproject/mantid/blob/master/scripts/Inelastic/CrystalField/pointcharge.py#L401). Beforehand, `np.unique` sees even the values that differed by 1e-17/1e-18 as different so many more entries were left around. Making more of these degenerate causes the shift.
@mducle Are you concerned about the changes? This test was only added very recently anyway and I assume the values were just what was given at the time.

Reply:
@martyngigg @AndreiSavici 

I think this is more an issue with the `.Neighbour` option in `PointCharge` rather than in `UnitCell` - this was put in as a convenience for users (the code actually uses the `.MaxDistance` property and just converts the nth neighbour in `.Neighbour` to an equivalent distance. My guess is that using `np.unique` is giving it the wrong `.MaxDistance`.

So I think the threshold in `UnitCell` is fine, but we should create a new issue about fixing `.Neighbour` calculation in `PointCharge`.

Reply:
@mducle Thanks. I think I have spotted another issue with the code I'm just not sure what the expected behaviour is. In [_calculateLigands](https://github.com/mantidproject/mantid/blob/master/scripts/Inelastic/CrystalField/pointcharge.py#L389) there is a loop over each position that accumulates the distances. There seems to be a scope issue as the `entries` list is defined inside the loop but used outside of it [below](https://github.com/mantidproject/mantid/blob/master/scripts/Inelastic/CrystalField/pointcharge.py#L402).

Is there an algorithm defined somewhere that lists how the ligand calculation is supposed to be performed?

Reply:
@martyngigg Doh! I took the algorithm from McPhase: https://github.com/mducle/mcphase/blob/master/bin/makenn.pl#L299 the original code is in perl :( 
There might be a better algorithm...